### ISSUE TITLE
nresolved compilation problems

### ISSUE THREAD

**Issue by lyingfromyou** [googleads/google-ads-java#94]
When I run GetAccountInformation, I am prompted with the following error：

    `Exception in thread "main" java.lang.Error: Unresolved compilation problems: 
	AutoValue_GoogleAdsClient cannot be resolved to a type
	AutoValue_GoogleAdsClient cannot be resolved to a type
	The method setInterceptorProvider(GrpcInterceptorProvider) in the type InstantiatingGrpcChannelProvider.Builder is not applicable for the arguments (() -> {})
	The method of(E) in the type ImmutableList is not applicable for the arguments (LoggingInterceptor)

	at com.google.ads.googleads.lib.GoogleAdsClient.newBuilder(GoogleAdsClient.java:69)
	at com.google.ads.googleads.examples.accountmanagement.GetAccountInformation.main(GetAccountInformation.java:52)
`
Cannot find AutoValue_GoogleAdsClient file，Is there any code not submitted?
![image](https://user-images.githubusercontent.com/24619864/56879445-5294e980-6a8b-11e9-9359-327506303e0f.png)

**Comment by nwbirnie** [comment]
Hey, this class is generated by annotation processing during compilation. If you build and run the examples from the command line with maven you'll find that this class is properly generated.

However, some IDEs circumvent the Maven build and run their own compiler chain (I believe that you're using Eclipse, but this also applies to IntelliJ). You need to configure the project compilation settings to enable the AutoValue annotation processor.

A quick Google for enabling AutoValue in Eclipse suggests that you navigate to Project properties > Java Compiler > Annotation Processing and add these Jars to the classpath:

- auto-service-1.0-rc1.jar
- guava-16.0.1.jar
- jsr-305-2.0.3.jar
- auto-value-1.0-rc1.jar

Can you try configuring the annotation processor and see if that fixes your issue please?

**Comment by iftfish** [comment]
I came across with the same problem and cannot fix by adding the above Jars manually.  
  
I solved it using the method in [this post](https://stackoverflow.com/a/48046613). First install m2e-apt plugin in eclipse market place. After installation, go to "project properties" -> Maven -> "Annotation Processing" -> "Enable project specific settings" -> "Automatically configure JDT APT". And this worked for me.
